include/deal.II/base/patterns.h

```diff
@@ -2384,6 +2388,64 @@ namespace Patterns
       }
     };
 
+#ifdef DEAL_II_WITH_MAGIC_ENUM
+    // Enums
```

Comment:
Is that comment necessary?

Reply:
It follows the structure of the rest of the file. Before each type of supported conversion (i.e., `// Tuples`, or `// Maps` , or `// Containers`) we have a similar one line comment.